app/models/task.py

```diff
@@ -2,4 +2,29 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
Same comment as above in Goal, I prefer the id attribute for the primary key of a model to just be `id`.